astropy/units/imperial.py

```diff
@@ -2,8 +2,15 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 
 """
-This package defines colloquially used Imperial units.  By default, they
-are not enabled.  To enable them, do::
+This package defines colloquially used Imperial units.  They are
+available in the `astropy.units.imperial` namespace, but not in the
+top-level `astropy.units` namespace, e.g.::
+
+    >>> import astropy.units as u
+    >>> mph = u.imperial.mile / u.hour  # doctest: +SKIP
```

Comment:
Oops, sorry, I thought this was the documentation, not the docstring. Still, perhaps fine to make the docstring pass the test too...